core/src/executor/evaluate/function.rs

```diff
@@ -763,3 +763,16 @@ pub fn length<'a>(name: String, expr: Evaluated<'_>) -> Result<Evaluated<'a>> {
         _ => Err(EvaluateError::FunctionRequiresStrOrListOrMapValue(name).into()),
     }
 }
+
+pub fn entries<'a>(name: String, expr: Evaluated<'_>) -> Result<Evaluated<'a>> {
+    match expr.try_into()? {
+        Value::Map(expr) => {
+            let entries = expr
+                .into_iter()
+                .map(|(k, v)| Value::List(vec![Value::Str(k), v]))
+                .collect::<Vec<_>>();
+            Ok(Evaluated::Value(Value::List(entries)))
+        }
+        _ => Err(EvaluateError::FunctionRequiresMapValue(name).into()),
+    }
```

Comment:
only one test left!
https://coveralls.io/builds/61509376/source?filename=core%2Fsrc%2Fexecutor%2Fevaluate%2Ffunction.rs#L775
```rust
test!(
        "SELECT UNWRAP('abc', 'a.b.c') FROM MapType",
    Err(EvaluateError::FunctionRequiresMapValue("UNWRAP".to_owned()).into())
    );
```

Reply:
thanks for the feedback. i've applied it :)